src/tests/tracing/eventpipe/simpleruntimeeventvalidation/simpleruntimeeventvalidation.cs

```diff
@@ -73,6 +82,21 @@ public static int Main()
             }
         };
 
+        private static Action _eventGeneratingActionForFinalizers = () =>
+        {
+            for (int i = 0; i < 50; i++)
+            {
+                if (i % 10 == 0)
+                    Logger.logger.Log($"Called GC.WaitForPendingFinalizers() {i} times...");
+                RuntimeEventValidation providerValidation = new RuntimeEventValidation();
+                providerValidation = null;
```

Comment:
Why is this needed for the test? Shouldn't just the `WaitForPendingFinalizers` call be sufficient `RuntimeEventValidation` doesn`t have a finalizer, so it shouldn't affect the count of finalizers run either.